setup/vld-setup.iss

```diff
@@ -2,11 +2,11 @@
 ; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
 
 #define MyAppName "Visual Leak Detector"
-#define MyAppVersion "2.5.6"
+#define MyAppVersion "2.5.7"
 #define MyAppPublisher "VLD Team"
 #define MyAppURL "http://vld.codeplex.com/"
 #define MyAppRegKey "Software\Visual Leak Detector"
-#define ConfigType "Release"
+#define ConfigType "Debug"

```

Comment:
>Debug [](start = 20, length = 5)

Probably should leave this checked in as Release by default.

Reply:
Yes, changed it back to Release

---
In reply to: [591982836](https://github.com/Azure/vld/pull/8#discussion_r591982836) [](ancestors = 591982836)